pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/define-control-plane-endpoint/index.md

```diff
@@ -8,7 +8,7 @@ beta: false
 enterprise: false
 ---
 
-The control plane should have three, five, or seven nodes, so that it can remain available if one, two, or three nodes fail, respectively. A control plane with one node should not be used in production.
+A control plane should have three, five, or seven nodes, so that it can remain available if one, two, or three nodes fail. A control plane with one node should not be used in production.
```

Comment:
```suggestion
A control plane should have three, five, or seven nodes, so it can remain available if one, two, or three nodes fail. A control plane with one node should not be used in production.
```